app/helpers/application_helper.rb

```diff
@@ -435,4 +435,11 @@ def hidden_spinner(text = '', options = {})
   def hosts_count(resource_name = controller.resource_name)
     @hosts_count ||= Host::Managed.reorder('').authorized.group("#{resource_name}_id").count
   end
+
+  def webpack_dev_server
+    return unless Rails.env.development?
+    host = ::Rails.configuration.webpack.dev_server.host.call
```

Comment:
I think you want to set this to 0.0.0.0 since it's not configurable anywhere - otherwise this will be `localhost` and people get  `Content Security Policy: The page's settings blocked the loading of a resource at http://localhost:3808/webpack/bundle.js ("script-src 'unsafe-eval' 'unsafe-inline' http://localhost:3000 http://0.0.0.0:3808").`. 

I just cloned the repo and started from fresh and that's the error you'd get everywhere without that change.